build.js

```diff
@@ -50,6 +62,10 @@ const buildOptions = {
   },
 };
 
+const rv = spawnSync("rustc", ["--version"]);
+console.log("rustc version stdout", rv.stdout.toString());
+console.log("rustc version stderr", rv.stderr.toString());
```

Comment:
looks like this and the `rust-toolchain.toml` change are the only non-formatting changes?

Reply:
That's right, yeah